freedesktop/com.github.blackhole89.notekit.metainfo.xml

```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+	<id>com.github.blackhole89.notekit.desktop</id>
+	<metadata_license>MIT</metadata_license>
+	<project_license>MIT</project_license>
+	<name>NoteKit</name>
+	<summary>A GTK3 hierarchical markdown notetaking application with tablet support</summary>
+	<description>
+		<p>
+			This program is a structured notetaking application based on GTK+ 3.
+			Write your notes in instantly-formatted Markdown, organise them in a
+			tree of folders that can be instantly navigated from within the
+			program, and add hand-drawn notes by mouse, touchscreen or
+			digitiser.
+		</p>
+		<p>
+			<em>Why?</em>
+		</p>
+		<p>
+			I figured it would be nice to have a free-software,
+			platform-independent OneNote. While there is a remarkable number of
+			free (speech or beer) notetaking applications out there, to my best
+			knowledge, none of them simultaneously check the following boxes:
+		</p>
+		<ul>
+			<li>note organisation</li>
+			<li>text as a first-class object</li>
+			<li>formatting</li>
+			<li>simple, standard on-disk format</li>
+			<li>tablet input</li>
+		</ul>
+		<p>
+			<em>Usage notes</em>
+		</p>
+		<p>
+			Note management
+		</p>
+		<ul>
+			<li>To create a new note, doubleclick a <code>+</code> node in the tree view and enter a name.</li>
+			<li>To create a new <em>folder</em>, doubleclick a <code>+</code> node in the tree view and enter a name ending in <code>/</code>, e.g. new <code>folder/</code>.</li>
```

Comment:
No you can't use <code> or any tags inside li. Despite the specs allowing them, none of the parser supports them correctly.

Reply:
Hum, quite an interesting problem... Is solving this planned for the future or is it at least planned to rewrite the specification accordingly?

Reply:
There are open issues about those yes